android/build.gradle

```diff
@@ -43,9 +44,13 @@ android {
         buildConfigField("String", "USER_AGENT_NAME", "\"Didomi Flutter SDK\"")
     }
 
+    buildFeatures {
+        buildConfig = true
+    }
+
     compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_1_8
-        targetCompatibility = JavaVersion.VERSION_1_8
+        sourceCompatibility = JavaVersion.VERSION_17
```

Comment:
Why do we need to change this, and is there an impact on clients apps?

Reply:
Yes, Android Studio is using JDK 17 by default. The client would have to do the same when updating the build configuration (AGP).